pkg/envoy/rds/response.go

```diff
@@ -1,8 +1,10 @@
 package rds
 
 import (
+	mapset "github.com/deckarep/golang-set"
 	xds_discovery "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
 	"github.com/envoyproxy/go-control-plane/pkg/cache/types"
+	"github.com/envoyproxy/go-control-plane/pkg/cache/v2"
```

Comment:
Would it make sense to use `v3` instead? I see that v2 is gone from main branch.

Reply:
I should have already fixed this. this is a miss-push.